src/bibliography.jl

```diff
@@ -20,15 +20,17 @@ abstract type ExpandBibliography <: Builder.DocumentPipeline end
 
 Selectors.order(::Type{ExpandBibliography}) = 2.12  # after CollectCitations
 
-function Selectors.runner(::Type{ExpandBibliography}, doc::Documents.Document)
+function Selectors.runner(::Type{ExpandBibliography}, doc::Documenter.Document)
     Documenter.Builder.is_doctest_only(doc, "ExpandBibliography") && return
     @info "ExpandBibliography: expanding `@bibliography` blocks."
+    global any_code = false
     for src in keys(doc.blueprint.pages)
         page = doc.blueprint.pages[src]
         empty!(page.globals.meta)
-        for element in page.elements
-            if Expanders.iscode(element, r"^@bibliography")
-                Selectors.dispatch(BibliographyBlock, element, page, doc)
+        for node in AbstractTrees.PreOrderDFS(page.mdast)
+            if node.element isa MarkdownAST.CodeBlock && occursin(r"^@bibliography", node.element.info)
+                # TODO: Why not call the method directly here?
+                Selectors.dispatch(BibliographyBlock, node, page, doc)
```

Comment:
Not quite sure what's going on here. Is `BibliographyBlock` and abstract `<: AbstractSelector` with multiple concrete implementations, each with their own `runner`s? It doesn't look to be.. so I am not sure why it would use the `Selectors` API here.